internal/options/options.go

```diff
@@ -4,6 +4,7 @@ type Options struct {
 	Type      string `name:"type" env:"HACKENV_TYPE" default:"kali" enum:"kali,parrot" help:"The VM to control with this command"`
 	Keymap    string `name:"keymap" env:"HACKENV_KEYMAP" default:"" help:"The keyboard keymap to force"`
 	Provision bool   `name:"provision" env:"HACKENV_PROVISION" help:"Provision the VM"`
+	Verbose   bool   `name:"verbose" help:"Verbose mode"`
```

Comment:
This flag could be enhanced with a `short:"v"` tag, so we can give it as `-v` in the command line.